### PR TITLE
Support Python 3.11

### PR DESCRIPTION
**Pull request by genestack-okunitsyn**

### CODE REVIEW

**Review by klas-genestack — CHANGES_REQUESTED** [review]
Update `classifiers` in `setup.py` as well, then =)

File: genestack_client/share_util.py [hunk 1]
```diff
@@ -13,6 +13,11 @@
 from genestack_client import Application
 
 
+try:
+    collectionsAbc = collections.abc
+except AttributeError:
+    collectionsAbc = collections
```
Comment (klas-genestack) [inline genestack_client/share_util.py:19]: Where is this import used?

Comment (genestack-okunitsyn) [reply genestack_client/share_util.py:18]: Autotests and publicDataLoader.

**Comment by genestack-okunitsyn** [comment]
> Update `classifiers` in `setup.py` as well, then =)

done.

**Review by tropnikovvl — APPROVED** [review]

**Review by klas-genestack — APPROVED** [review]

**Pull request merged**